releases/23.0.0.12/beta/Dockerfile.ubi.openjdk17

```diff
@@ -0,0 +1,153 @@
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-jdk-ubi AS getRuntime
```

Comment:
`beta` is only needed for `latest`. Remove folder `releases/23.0.0.12/beta`

Reply:
I definitely need to fix script file so that it removes the beta directory from new release directory, which it does by making a copy of the "latest" dir.